tests/test-solid-solid-collision.cc

```diff
@@ -89,14 +80,14 @@ BOOST_AUTO_TEST_CASE (solid_solid_collision_1)
   display(model, bpc->coefficients());
 
   ConstObjectStdVector_t obstacles;
-  fcl::CollisionGeometryPtr_t box (new fcl::Box (.2, .4, .6));
+  hpp::fcl::CollisionGeometryPtr_t box (new hpp::fcl::Box (.2, .4, .6));
```

Comment:
This is probably a bit early, as at the moment fcl is not yet in the hpp namespace

Reply:
This goes with https://github.com/humanoid-path-planner/hpp-fcl/pull/56 which is the next devel version.

Reply:
@jmirabel  does this mean that we have to merge simultaneously pinocchiov2 and the namespace change of hpp-fcl ????

Reply:
That was the idea, yes. Do you see a problem with this ? The namespace change shouldn't impact your code much, if it even impact it.

Reply:
@stonneau cf discussions in https://github.com/stack-of-tasks/pinocchio/pull/645

Reply:
I am very confused right now, because obviously in the other PR the change has not been made. At least for pinocchio. All the PRs related to pinocchiov2 compile without the change of namespace, so what am I supposed to do ? make the switch or first allow for an intial merge without the namespace renaming?

Reply:
@jmirabel  some prs are updated to hpp::fcl namespace, such as hpp-corbaserver, some others are not. We need to choose what to do now, and I recommend that we first make a pull request prior to the namespace change

Reply:
> some others are not

Which one ?

> We need to choose what to do now, and I recommend that we first make a pull request prior to the namespace change

So far, the namespace change has only minimal impact and I don't think it will have more impact. Did you encounter an unforeseen one ?
I won't do the work of removing the commits right now that correct the changes because I am busy with other matters and because I don't think it's worth the effort.

Reply:
Well as I said at least pinocchio

Reply:
I think Pinocchio works fine (assuming the PR) with hpp-fcl in hpp::fcl.

Reply:
My bad, for some reasons when I checked the files from the pr the namespace commit was missing. I apologize for that